### ISSUE TITLE
not showing issue/pr title, just its number

### ISSUE THREAD

**Issue by gnfisher** [nathonius/obsidian-github-link#83]
![Image](https://github.com/nathonius/obsidian-github-link/assets/4742306/4d1d75ea-ff94-47e0-878a-dc10fe77f986)

In the examples here the name of the issue/PR is shown. But I am only seeing the issue number. Am I missing a config with something? Thanks!

**Comment by nathonius** [comment]
@gnfisher it will only show the number if there's something preventing hitting the GitHub API. Can you enable debug logging in plugin settings and see if there are any console errors or anything? (ctrl+shift+i to open the devtools)

Also, are you using a token? If so, what are you using for the value of the "orgs" field? And how was the token generated?

**Comment by gnfisher** [comment]
- Its a PAT
- I just added `read:org` and now public repos are showing the titles, by private org owned (for work) repos are not.
- Debug, but nothing in the console.

**Comment by gnfisher** [comment]
I think the issue is that the org I want to see item information for requires SAML auth on GitHub, so it's no dice. Public stuff in stuff and my private repos seem to be working after I added `read:org` permissions and deleted the links and rewrote them.

I'm using a PAT.

**Comment by gnfisher** [comment]
Ah ha, had to SSO authorize my PAT, too. It's working great, thanks so much for working on this, it's fantastic!

**Comment by nathonius** [comment]
@gnfisher Yeah, auth is a bit of a bear with this; the best way would be for each user to create their own OAuth app, but that asks a lot of people who may just want to see github links. 👍 Glad you got it figured out! If you want and can describe the issue/process you went through, I can get some more notes added to the documentation.
